proto/metrics/metrics_service.proto

```diff
@@ -0,0 +1,16 @@
+syntax = "proto3";
+
+package metrics;
+
+import "metrics/metrics.proto";
+import "google/protobuf/empty.proto";
+
+option go_package = "metrics-and-alerting-service/proto/metrics";
+
+service Metrics {
+  rpc UpdateMetric(UpdateMetricRequest) returns (UpdateMetricResponse);
+  rpc UpdateMetricsBatch(UpdateMetricsBatchRequest) returns (google.protobuf.Empty);
+  rpc GetMetric(GetMetricRequest) returns (GetMetricResponse);
+  rpc GetMetrics(google.protobuf.Empty) returns (GetMetricsResponse);
+  rpc Ping(google.protobuf.Empty) returns (google.protobuf.Empty);
```

Comment:
If you want, you can try [grpc-gateway](https://github.com/grpc-ecosystem/grpc-gateway)